README.rst

```diff
@@ -176,22 +176,22 @@ The ``WebLogAnalyticsApp`` can be built and packaged using the Apache Maven comm
 
   $ mvn clean package
 
-Note that the remaining commands assume that the ``cdap-cli.sh`` script is
+Note that the remaining commands assume that the ``cdap`` script is
 available on your PATH. If this is not the case, please add it::
 
   $ export PATH=$PATH:<CDAP home>/bin
 
 If you haven't already started a standalone CDAP installation, start it with the command::
 
-  $ cdap.sh start
+  $ cdap sdk start
 
 We can then deploy the application to a standalone CDAP installation and
 start the flow and service::
 
-  $ cdap-cli.sh -u localhost:10000/default load artifact target/cdap-flume-guide-<version>.jar
-  $ cdap-cli.sh -u localhost:10000/default create app WebLogAnalyticsApp cdap-flume-guide <version> user
-  $ cdap-cli.sh -u localhost:10000/default start flow WebLogAnalyticsApp.WebLogAnalyticsFlow
-  $ cdap-cli.sh -u localhost:10000/default start service WebLogAnalyticsApp.WebLogAnalyticsService
+  $ cdap cli -u localhost:10000/default load artifact target/cdap-flume-guide-<version>.jar
+  $ cdap cli -u localhost:10000/default create app WebLogAnalyticsApp cdap-flume-guide <version> user
+  $ cdap cli -u localhost:10000/default start flow WebLogAnalyticsApp.WebLogAnalyticsFlow
+  $ cdap cli -u localhost:10000/default start service WebLogAnalyticsApp.WebLogAnalyticsService
```

Comment:
Wrong port and `-u localhost:correct_port/default` is the default. I would just remove all of the -u options.